src/corehost/src/args.cpp

```diff
@@ -70,23 +82,8 @@ bool parse_arguments(const int argc, const pal::char_t* argv[], arguments_t& arg
         }
     }
 
-    // Read CLR path from environment variable
-    pal::string_t home_str;
-    pal::getenv(_X("DOTNET_HOME"), home_str);
-    if (!home_str.empty())
-    {
-        append_path(home_str, _X("runtime"));
-        append_path(home_str, _X("coreclr"));
-        args.clr_path.assign(home_str);
-    }
-    else
-    {
-        // Use platform-specific search algorithm
-        if (pal::find_coreclr(home_str))
-        {
-            args.clr_path.assign(home_str);
-        }
-    }
-
+    pal::getenv(_X("DOTNET_SERVICING"), args.svc_dir);
+    pal::getenv(_X("DOTNET_RUNTIME_SERVICING"), args.runtime_svc_dir);
+    pal::getenv(_X("DOTNET_HOME"), args.home_dir);
```

Comment:
Good catch. I had a Ctrl+X in the buffer and got interrupted and forgot about it. Thanks for pointing out!